### PR TITLE
Roll dice command

### PR DESCRIPTION
**Pull request by EdwinYi-JanYang**
# Related Issues
Resolves Issue #95 

# Summary of Changes 
Adds command .rd n which rolls an n-sided dice and pings the user with the returned value.

# Steps to Reproduce

### CODE REVIEW

File: src/commands/fun/rollDice.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+class RollDiceCommand extends BaseCommand {
+  constructor(client: CommandoClient) {
+    super(client, {
+      name: 'roll-dice',
+      aliases: ['rd', 'roll', 'rolldice', 'dice-roll', 'diceroll', 'dice'],
+      group: 'fun',
+      args: [
+        {
+          key: 'sides',
```
Comment (gzcharleszhang) [inline src/commands/fun/rollDice.ts:13]: Can default to 6 sides

File: src/commands/fun/rollDice.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+          type: 'integer'
+        }
+      ],
+      memberName: 'dice-coin',
+      description: 'Roll a dice!',
+      examples: [`${client.commandPrefix}dice-roll`]
+    });
+  }
+
+  getRandomInt(max: number): number {
```
Comment (gzcharleszhang) [inline src/commands/fun/rollDice.ts:24]: I think this gives you [1, max) instead of [1, max]

File: src/commands/fun/rollDice.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    }
+    return roll;
+  }
+
+  async onRun(message: CommandoMessage, args: {sides: number;}): Promise<Message> {
+    const {sides} = args;
+    if (sides <= 0) {
+      return message.reply(`I cannot compute ` + sides + ' sides!');
+    }
+    else if (sides > 1000000) {
```
Comment (gzcharleszhang) [inline src/commands/fun/rollDice.ts:37]: Can change else if to an if since the previous of statement exits the function

**Review by gzcharleszhang — CHANGES_REQUESTED** [review]

File: src/commands/fun/rollDice.ts (hunk 1 above)
Comment (EdwinYi-JanYang) [reply src/commands/fun/rollDice.ts:13]: how would I default to 6 sides?

File: src/commands/fun/rollDice.ts (hunk 2 above)
Comment (EdwinYi-JanYang) [reply src/commands/fun/rollDice.ts:24]: yes this is a problem

File: src/commands/fun/rollDice.ts (hunk 3 above)
Comment (EdwinYi-JanYang) [reply src/commands/fun/rollDice.ts:37]: understood

File: src/commands/fun/rollDice.ts (hunk 2 above)
Comment (EdwinYi-JanYang) [reply src/commands/fun/rollDice.ts:24]: it should be fixed now

File: src/commands/fun/rollDice.ts (hunk 1 above)
Comment (gzcharleszhang) [reply src/commands/fun/rollDice.ts:13]: Try adding default: 6

Comment (EdwinYi-JanYang) [reply src/commands/fun/rollDice.ts:13]: alright

File: src/commands/fun/rollDice.ts (hunk 2 above)
Comment (gzcharleszhang) [reply src/commands/fun/rollDice.ts:24]: This is still not quite right, the probability of rolling a 1 is higher than all other options

Comment (EdwinYi-JanYang) [reply src/commands/fun/rollDice.ts:25]: This is very bad

**Review by gzcharleszhang — APPROVED** [review]

**Pull request merged**